pinc/user_is.inc

```diff
@@ -139,7 +139,7 @@ function that_user_is_over_PP_checked_out_limit($username)
         SELECT COUNT(*)
         FROM projects
         WHERE checkedoutby LIKE '%s'
-            AND state = 'proj_post_first_checked_out'
+            AND state = '" . PROJ_POST_FIRST_CHECKED_OUT . "'
```

Comment:
We're already in an `sprintf()` here, let's just make this another param we pass it. Ditto some of the others here.

Reply:
Done this.